### PR TITLE
[CORL-163] Add feature button to moderation cards

### PR DESCRIPTION
**Pull request by nick-funk**
## What does this PR do?

Adds functionality for featuring and un-featuring comments from the moderation queue.
There is a new button to toggle featured state on a comment.

## How do I test this PR?

- Comment on a story
- Head to your moderation queue
- Notice that the story has a new button in top right labelled "Feature"
- Click it to approve/feature the comment
- Check your history and head to single moderation card view
- You can un-feature the comment if you would like
- Feature button should toggle accordingly with appropriate text and styles (i.e. switch to "Feature" vs "Featured" and colouring should change)

### CODE REVIEW

File: src/core/client/admin/test/moderate/feature.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  settings,
+  unmoderatedComments,
+  users,
+} from "../fixtures";
+
+const viewer = users.admins[0];
+
+beforeEach(async () => {
+  replaceHistoryLocation("http://localhost/admin/moderate");
+});
```
Comment (alinhle) [inline src/core/client/admin/test/moderate/feature.spec.tsx]: This location is always replaced to a different location in your tests, maybe we don't need these 3 lines then?

File: src/core/client/admin/routes/Moderate/ModerateCard/ModerateCard.tsx [hunk 2]
```diff
@@ -23,6 +24,7 @@ interface Props {
   inReplyTo: string | null;
   comment: PropTypesOf<typeof MarkersContainer>["comment"];
   status: "approved" | "rejected" | "undecided";
+  featured: true | false;
```
Comment (alinhle) [inline src/core/client/admin/routes/Moderate/ModerateCard/ModerateCard.tsx]: That's a `boolean` right? :-)

File: src/core/client/admin/test/moderate/feature.spec.tsx (hunk 1 above)
Comment (nick-funk) [reply src/core/client/admin/test/moderate/feature.spec.tsx]: Fixed

File: src/core/client/admin/routes/Moderate/ModerateCard/ModerateCard.tsx (hunk 2 above)
Comment (nick-funk) [reply src/core/client/admin/routes/Moderate/ModerateCard/ModerateCard.tsx]: You know it. Fixed.

File: src/core/server/graph/tenant/resolvers/FeatureCommentPayload.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { GQLFeatureCommentPayloadTypeResolver } from "coral-server/graph/tenant/schema/__generated__/types";
+
+import { moderationQueuesResolver } from "./ModerationQueues";
+
+export const FeatureCommentPayload: GQLFeatureCommentPayloadTypeResolver = {
+  moderationQueues: moderationQueuesResolver,
```
Comment (alinhle) [inline src/core/server/graph/tenant/resolvers/FeatureCommentPayload.ts:6]: 👍

**Review by alinhle — APPROVED** [review]
This is fabulous! Thank you @nick-funk !

**Pull request merged**